language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/codeaction/CodeActionTest.java

```diff
@@ -238,7 +238,7 @@ public void testCodeActionWithQuickFixes(String config, String source)
         log.info("Test textDocument/codeAction QuickFixes");
         return new Object[][]{
                 {"fixReturnType1.json", "fixReturnType.bal"},
-                {"fixReturnType2.json", "fixReturnType.bal"},
+//                {"fixReturnType2.json", "fixReturnType.bal"},
```

Comment:
Please create an issue for this and assign @nadeeshaan

Reply:
Yeah, I've created https://github.com/ballerina-platform/ballerina-lang/issues/21509.